bolt-modules/boltlib/spec/functions/puppetdb_fact.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'puppetdb_fact' do
+  include PuppetlabsSpec::Fixtures
+
+  let(:executor) { mock('bolt_executor') }
+  let(:inventory) { mock('inventory') }
+
+  around(:each) do |example|
+    Puppet[:tasks] = true
+    Puppet.features.stubs(:bolt?).returns(true)
+
+    Puppet.override(bolt_executor: executor, bolt_inventory: inventory) do
+      example.run
+    end
```

Comment:
This could use actual spec tests. Which probably require mocking PuppetDB::Client or the executor. So mostly not useful...

Reply:
I wrote a spec test and determined it wasn't really testing anything? But if you have ideas for a useful one let me know. Or I can add a simple one.

Reply:
I guess a simple one is useful to make sure there are no syntax errors.